migration-from-4x.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: default
+title: Migrating from TinyMCE 4.x to TinyMCE 5.0.
+title_nav: Migrating from 4.x
+description: Instructions for migrating from TinyMCE 4.x to TinyMCE 5.0.
+keywords: migration considerations premigration pre-migration
+---
+
+## Migrating from 4.x
+
+The new TinyMCE 5.0 editor comes with significant changes to the previous versions, though most configuration changes affect complex use cases like adding custom components. Setting up a basic TinyMCE 5.0 instance should be no harder than with TinyMCE 4.x.
+
+Our team at Tiny has worked on creating a more configurable, more modern and less cumbersome editor while retaining the familiarity of the user interface from the older versions.
+
+This chapter describes the migration process and workarounds if you are using an older version of TinyMCE. It includes tasks that you must perform before the migration can begin, and different workaround procedures for deprecated, deleted, and updated features.
+
+## Editor-Core
+
+### Initialization
+
+The initialization process of TinyMCE 5.0 is the same as TinyMCE 4.x. The bootstrap process and initialization events all remain the same.  The main differences are in the `init` configuration, specifically the configuration items for UI components. It still retains a familiar JSON structure. However, many of the configuration options have been simplified.
+
+### Settings
+
+In TinyMCE 5.0, some configurations have been removed because they are no longer necessary or an improved solution has been introduced.  All inline style configurations have been removed in TinyMCE 5.0 in favour of modern CSS.
+
+#### Changed Settings
+
+| **Old Settings** | **Description** | **Alternative** |
```

Comment:
Could we add a blurb to say where these settings came from in TinyMCE 4.x? It's a bit confusing at the moment, to just have a list of what looks like CSS styles...

Reply:
Also, should these go under Removed Settings rather than Changed Settings?

Reply:
yes will re-edit the text for clarity

yeah thats true, they are a removed setting. We should move this to a removed settings

Reply:
Fixed